src/testmod/java/ladysnake/satintestcore/item/DebugItem.java

```diff
@@ -41,14 +45,19 @@ public void registerDebugCallback(DebugCallback callback) {
         this.callbacks.add(callback);
     }
 
+    public void registerDebugCallback(String name, DebugCallback callback) {
+        this.callbacks.add(callback);
+        this.callbackNames.put(callback, name);
+    }
+
     @Override
     public TypedActionResult<ItemStack> use(World world, PlayerEntity player, Hand hand) {
         if (player.isSneaking() && !world.isClient) {
             if (this.callbacks.size() > 1) {
                 debugMode = (debugMode + 1) % this.callbacks.size();
-                player.sendMessage(Text.translatable("Switched mode to %s", debugMode), true);
+                player.sendMessage(Text.translatable("Switched mode to %s", this.callbackNames.getOrDefault(this.callbacks.get(debugMode), ""+debugMode)), true);
```

Comment:
```suggestion
                player.sendMessage(Text.translatable("Switched mode to %s", this.callbackNames.getOrDefault(this.callbacks.get(debugMode), String.valueOf(debugMode))), true);
```
Just a style preference